src/app/components/Validators/index.tsx

```diff
@@ -0,0 +1,167 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { styled } from '@mui/material/styles'
+import { Table, TableCellAlign, TableColProps } from '../../components/Table'
+import { Validator } from '../../../oasis-nexus/api'
+import { TablePaginationProps } from '../Table/TablePagination'
+import { StatusIcon } from '../StatusIcon'
+import { RoundedBalance } from '../RoundedBalance'
+
+type ValidatorsProps = {
+  validators?: Validator[]
+  isLoading: boolean
+  limit: number
+  pagination: false | TablePaginationProps
+}
+
+const StyledImage = styled('img')({
+  width: '28px',
+  height: '28px',
+  borderRadius: 5,
+  marginRight: 15,
+})
+
+const StyledBox = styled(Box, {
+  shouldForwardProp: prop => prop !== 'value',
+})<{ value: number }>(({ value, theme }) => ({
+  position: 'relative',
+  textAlign: 'center',
+  '&::before': {
+    content: '""',
```

Comment:
suspicious

Reply:
content: '',
`Error: You seem to be using a value for 'content' without quotes, try replacing it with `content: '""`